method/application/accept.js

```diff
@@ -0,0 +1,34 @@
+const applicationSpec = require('../../spec/application/application')
+
+module.exports = function AcceptGroupApplication (server) {
+  return function acceptGroupApplication (applicationId, text, cb) {
+    server.tribes.application.get(applicationId, (getErr, applicationData) => {
+      const { groupId, recps } = applicationData
+      server.tribes.invite(groupId, recps, { text }, (err, invite) => {
+        const content = {
+          type: 'group/application',
+          version: 'v1',
+          addMember: {
+            add: invite.key
+          },
+          text: {
+            append: text
+          },
+          recps,
+          tangles: {
+            application: {
+              root: applicationId,
+              previous: [applicationData.key]
+            }
+          }
+        }
+        if (!applicationSpec.isValid(content)) {
+          return cb(applicationSpec.isValid.errors)
+        }
+        server.publish(content, (_, publishData) => {
```

Comment:
this is throwing out errors, please use `if (err) return cb(err)`

Reply:
But this is a spec validation error, doesn't follow this pattern, does it?

Reply:
I'm guessing ur refering to `server.publish(content, (_, publishData) => {` correct?

If so got it. Noticiced `return` in this case is irrelevant.

Reply:
cool